eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -401,10 +401,10 @@ the main ServiceBusClientBuilder. -->
   <!-- perf-test files are not public API, so they don't need to fulfill all requirements. -->
   <suppress checks="Javadoc" files=".*[/\\]azure-storage-perf|azure-search-perf|azure-identity-perf|azure-template-perf[/\\].*\.java"/>
   <suppress checks="com.azure.tools.checkstyle.checks.EnforceFinalFields"
-            files=".*[/\\]azure-identity-perf|azure-search-perf[/\\].*\.java"/>
-  <suppress checks="GoodLogging" files=".*[/\\]azure-storage-perf|azure-search-perf|perf-test-core[/\\].*\.java"/>
+            files=".*[/\\]azure-identity-perf|azure-search-perf|azure-ai-formrecognizer-perf[/\\].*\.java"/>
+  <suppress checks="GoodLogging" files=".*[/\\]azure-storage-perf|azure-search-perf|azure-ai-formrecognizer-perf|perf-test-core[/\\].*\.java"/>
```

Comment:
Do we really need those checkstyle suppressions. I only found a few that can be fixed in TA. If need, just ignored my comment. ;>